spec/schema.json

```diff
@@ -540,20 +540,43 @@
         }
       },
       "BeneficialOwnerIndividual": {
+        "required": [
+          "first_name",
+          "last_name",
+          "date_of_birth",
+          "nationality",
+          "is_service_address_same_as_usual_residential_address",
+          "start_date",
+          "is_on_sanctions_list"
```

Comment:
`usual_residential_address` should be included, as well of of the NOC. For the second one we can do something similar to the  `components/schemas/Address`

Reply:
We need to add something like
```
"required": [
  "first_name",
  "last_name",
  "date_of_birth",
  "nationality",
  "usual_residential_address",
  "is_service_address_same_as_usual_residential_address",
  "start_date",
  "is_on_sanctions_list"
],
"anyOf": [
  {
    "required": [
      "beneficial_owner_nature_of_control_types"
    ]
  },
  {
    "required": [
      "trustees_nature_of_control_types",
      "trust_ids"
    ]
  },
  {
    "required": [
      "non_legal_firm_members_nature_of_control_types"
    ]
  }
],
```

Reply:
Ok, `usual_residential_address` now included.  NOC not as it's still unclear how this can be defined (if at all). Suggestion is to merge this now and follow up with spec changes across the different data blocks if we can work out how to do it.

Reply:
Thanks @mwestacott, it looks like there is a bug in the swagger Editor UI, so I will approve it anyway. But syntactically it should be correct the previous snippet or the following 
```
"anyOf": [
  {
    "required": [
      "first_name",
      "last_name",
      "date_of_birth",
      "nationality",
      "usual_residential_address",
      "is_service_address_same_as_usual_residential_address",
      "start_date",
      "is_on_sanctions_list"
    ],
  },
  {
    "required": [
      "beneficial_owner_nature_of_control_types"
    ]
  },
  {
    "required": [
      "trustees_nature_of_control_types",
      "trust_ids"
    ]
  },
  {
    "required": [
      "non_legal_firm_members_nature_of_control_types"
    ]
  }
```